tests/IBFE/spread_01.cpp

```diff
@@ -0,0 +1,336 @@
+        VariableDatabase<NDIM>* var_db = VariableDatabase<NDIM>::getDatabase();
+        const Pointer<SAMRAI::hier::Variable<NDIM> > f_var = time_integrator->getBodyForceVariable();
+        const Pointer<VariableContext> f_ghost_ctx = var_db->getContext("f_ghost");
+        const int f_ghost_idx = var_db->registerVariableAndContext(f_var, f_ghost_ctx, 3);
+
+        for (int ln = 0; ln <= patch_hierarchy->getFinestLevelNumber(); ++ln)
+        {
+            Pointer<PatchLevel<NDIM> > level = patch_hierarchy->getPatchLevel(ln);
+            level->allocatePatchData(f_ghost_idx);
+            for (PatchLevel<NDIM>::Iterator p(level); p; p++)
+            {
+                Pointer<Patch<NDIM> > patch = level->getPatch(p());
+                Pointer<SideData<NDIM, double> > f_data = patch->getPatchData(f_ghost_idx);
+                f_data->fillAll(0.0);
+            }
+        }
+
+        // synch ghost data
+        using InterpolationTransactionComponent = HierarchyGhostCellInterpolation::InterpolationTransactionComponent;
+        std::vector<InterpolationTransactionComponent> ghost_cell_components(1);
+        ghost_cell_components[0] = InterpolationTransactionComponent(f_ghost_idx,
+                                                                     "CONSERVATIVE_LINEAR_REFINE",
+                                                                     true,
+                                                                     "CONSERVATIVE_COARSEN",
+                                                                     "LINEAR",
+                                                                     false,
+                                                                     {}, // f_bc_coefs
+                                                                     nullptr);
+        HierarchyGhostCellInterpolation ghost_fill_op;
+        ghost_fill_op.initializeOperatorState(ghost_cell_components, patch_hierarchy);
+        ghost_fill_op.fillData(/*time*/0.0);
+
+        const double dt = time_integrator->getMaximumTimeStepSize();
+        time_integrator->preprocessIntegrateHierarchy(time_integrator->getIntegratorTime(),
+                                                      time_integrator->getIntegratorTime() + dt,
+                                                      1/*???*/);
+        auto &fe_data_manager = *ib_method_ops->getFEDataManager();
+        auto &equation_systems = *fe_data_manager.getEquationSystems();
+        auto &force_system = equation_systems.get_system(IBFEMethod::FORCE_SYSTEM_NAME);
+        auto &half_f_vector = dynamic_cast<libMesh::PetscVector<double> &>(*force_system.current_local_solution);
+        for (unsigned int i = half_f_vector.first_local_index(); i <= half_f_vector.last_local_index(); ++i)
+        {
+            half_f_vector.set(i, i % 10);
+        }
+        half_f_vector.close();
+
+        ib_method_ops->spreadForce(f_ghost_idx, nullptr, {}, time_integrator->getIntegratorTime() + dt/2);
```

Comment:
Its not clear to me how to compute convergence or errors for a spread force, so I just print out the values in this test.